app/spec/lib/bk/compat/__snapshots__/examples/github-actions/basic.yaml.snap

```diff
@@ -7,3 +7,4 @@ steps:
     runs-on: ubuntu-latest
   label: ":github: explore"
   key: explore
+  branches: ''
```

Comment:
It doesn't seem to be handling null? Shouldn't we _not_ have the filter if one isn't set?

Reply:
Good catch. I did try different scenarios and this has regressed in the last changes I made.

Reply:
@mcncl , fixed in latest commit